cypress/e2e/workspace.test.ts

```diff
@@ -36,10 +36,7 @@ context("Test the overall app", () => {
     {mode: Modes.simulation, organ: Organs.nose, title: "Immune Response Model Simulator"},
     {mode: Modes.simulation, organ: Organs.brain, title: "Focus Model Simulator"},
   ];
-  const modePages: PageInfo[] = [
-    {mode: Modes.animation, organ: Organs.heart, title: "Plaque Animation"},
-    {mode: Modes.simulation, organ: Organs.heart, title: "Plaque Model Simulator"}
-  ];
```

Comment:
This was probably removed automatically, but it's being used again in one of my currently open PRs, so this change will cause a merge conflict.